src/main/java/games/strategy/engine/framework/ui/background/WaitWindow.java

```diff
@@ -32,29 +41,27 @@ public void run() {
       }
     };
 
-    synchronized (m_mutex) {
-      if (m_timer != null) {
-        m_timer.schedule(task, 15, 15);
+    synchronized (mutex) {
+      if (timer != null) {
+        timer.schedule(task, 15, 15);
       }
     }
   }
 
+  /**
+   * Hides the wait window.
+   */
   public void doneWait() {
-    synchronized (m_mutex) {
-      if (m_timer != null) {
-        m_timer.cancel();
-        m_timer = null;
+    synchronized (mutex) {
+      if (timer != null) {
+        timer.cancel();
+        timer = null;
```

Comment:
:+1: Agreed and done in a1331aa.  I tested this on Linux, and it exhibits the desired behavior.